test/e2e/specs/editor/various/compatibility-classic-editor.spec.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Compatibility with classic editor', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test.afterEach( async ( { requestUtils } ) => {
+		await requestUtils.deleteAllPosts();
+	} );
+
+	test( 'Should not apply auto when rendering blocks', async ( {
+		page,
+		editor,
+	} ) => {
+		await editor.insertBlock( { name: 'core/html' } );
+		await page.focus( 'role=textbox[name="HTML"i]' );
+		await page.keyboard.type( '<a>' );
+		await page.keyboard.type( 'Random Link' );
+		await page.keyboard.type( '</a> ' );
+		// Publish Post
+		await editor.publishPost();
+		// View Post
+		await page.locator( 'role=link[name="View post"i]' ).first().click();
```

Comment:
Avoid using `first()`, trying chaining selectors instead.

```suggestion
		await page.click( 'role=region[name="Editor publish"i] >> role=link[name="View post"i]' );
```

Reply:
Addressed. Thanks